li-utils/src/main/pegasus/com/linkedin/common/AuditStamp.pdl

```diff
@@ -19,4 +19,9 @@ record AuditStamp {
    * The entity (e.g. a service URN) which performs the change on behalf of the Actor and must be authorized to act as the Actor.
    */
   impersonator: optional Urn
+
+  /**
+   * Additional message to keep track of in the event
```

Comment:
what are the sort of things that we would expect to be written into this `message` field for AuditStamp?

Reply:
The doc string is not super clear about what the purpose of this field is

Reply:
Agreed